### DISCUSSION TITLE
Can't reapply on the reapply page for student pack

### DISCUSSION THREAD

**Discussion by MateSandor** [community/community#119639] [GitHub Education]
### Hi everyone,

I got access to the student benefits around 2022. Now my access has been removed, but I still study. I want to reapply but when I upload the proof (my student dated Id) well first of all it's a two sided ID and I can only upload one picture. 

But the main problem is that if I press apply it just refreshes the page without anything happening.

I got a valid ID can someone help me?

**Comment by samus-aran** [comment]
Hi @MateSandor 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
